### DISCUSSION TITLE
Docker Image Experience for Running Gaphor on Linux

### DISCUSSION THREAD

**Discussion by jashangills** [gaphor/gaphor#3218] [Q&A]
Has anyone in the community had experience using Docker images for running Gaphor on Linux systems? I'm interested in hearing about your experiences, any challenges faced, and tips for setting up and using Gaphor with Docker. Your insights would be greatly appreciated!

**Comment by danyeaw** [comment]
Hi @jashangills, I don't have any experience with this exactly, but if you go to the .devcontainer folder in the repository, there is a Dockerfile for use with GitHub Codespaces. This could at least be a starting place.
